### DISCUSSION TITLE
basic example code dont work

### DISCUSSION THREAD

**Discussion by LAGENCECREE** [alpinejs/alpine#3869] [7. Bugs (archived)] [answered]
1. open a codepen or jsfiddle

2. paste 
```
<script>console.clear()</script>
<script src="//unpkg.com/alpinejs" defer></script>

<div x-init="$refs.button.remove()">
  <button x-ref="button">Remove Me</button>
</div>

<div>
  Copyright ©
 
  <span x-text="new Date().getFullYear()"></span>
</div>
```

3. see the console error 
alpinejs:5 Uncaught TypeError: Cannot read properties of undefined (reading '_x_refs')

-> https://jsfiddle.net/mLr75cnz/1/

**Comment by SimoTod** [comment, marked as answer]
There's no x-data in your example. You need x-data to define an Alpine component

**Reply by LAGENCECREE** [reply]
omg sorry, got it ! it was unclear with the given examples of the "Simple.Lightweight.Powerful as hell." section on https://alpinejs.dev/ page. many thanks

**Reply by SimoTod** [reply]
No worries, you are not the first asking this question. I think it says that in the docs but not on the very first page
